# CVE-2021-46823


## Metadata
| Published | Last Modified | Assigner | Version | Format |
| --------- | ------------- | -------- | ------- | ------ |
| 2022-06-18 16:15:00 | 2022-06-28 16:11:00 | cve@mitre.org | 4.0 | MITRE | 

## Description
| Language | Description |
| -------- | ----------- |
| en | python-ldap before 3.4.0 is vulnerable to a denial of service when ldap.schema is used for untrusted schema definitions, because of a regular expression denial of service (ReDoS) flaw in the LDAP schema parser. By sending crafted regex input, a remote authenticated attacker could exploit this vulnerability to cause a denial of service condition. |

## Configurations
| Vulnerable | CPE 23 URI | CPE Name | Operator |
| ---------- | ---------- | -------- | -------- |
| True | cpe:2.3:a:python-ldap:python-ldap:*:*:*:*:*:*:*:* | 3.4.0 | [] | OR |

## CVSSv3 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Attack Vector | Attack Complexity | Privileges Required | User Interaction | Scope | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | ------------------- | ---------------- | ----- | ---------------------- | ---------------- | ------------------- |
| 6.5 | MEDIUM | 3.6 | 2.8 | CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H | 3.1 | NETWORK | LOW | LOW | NONE | UNCHANGED | NONE | NONE | HIGH |

## CVSSv2 Information
| Score | Severity | Impact Score | Exploitability Score | Vector String | Version | Access Vector | Access Complexity | Authentication | User Interaction Required | Obtain All Privilge | Obtain User Privilege | Obtain Other Privilege | Confidentiality Impact | Integrity Impact | Availability Impact |
| ----- | -------- | ------------ | -------------------- | ------------- | ------- | ------------- | ----------------- | -------------- | ------------------------- | ------------------- | --------------------- | ---------------------- | ---------------------- | ---------------- | ------------------- |
| 4.0 | MEDIUM | 2.9 | 8.0 | AV:N/AC:L/Au:S/C:N/I:N/A:P | 2.0 | NETWORK | LOW | SINGLE | 0 | False | False | False | NONE | NONE | PARTIAL |

## Problem Type
NVD-CWE-Other

## References
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://github.com/python-ldap/python-ldap/security/advisories/GHSA-r8wq-qrxc-hmcm|https://github.com/python-ldap/python-ldap/security/advisories/GHSA-r8wq-qrxc-hmcm|MISC|['Third Party Advisory']
| Name | Reference URL | Reference Source | Tags |
| ---- | ------------- | ---------------- | ---- |
https://exchange.xforce.ibmcloud.com/vulnerabilities/221507|https://exchange.xforce.ibmcloud.com/vulnerabilities/221507|MISC|['Third Party Advisory', 'VDB Entry']
